lib/tasks/mysql_connection_helper.rb

```diff
@@ -0,0 +1,58 @@
+require 'mysql2'
+require 'yaml'
+
+# Require this on your rake tasks and everything will be fine!
+# Use require Report::MysqlConnection and use reports_client as well.
+module Report
```

Comment:
This module should be an open source code @ devlandia... it is really good. It needs one refactor here and there so we can use dynamic connections, but it is great as it is.